### ISSUE TITLE
Exporter crashes with index out of range

### ISSUE THREAD

**Issue by Kamisoi** [dirien/minecraft-prometheus-exporter#273]
### What happened?

Hello,
while trying to use minecraft-exporter for docker-hosted modded [modpack: Multiblock Madness, hosted via Pterodactyl Panel] Minecraft server I've encountered following error while scraping from Prometheus hosted on another machine:

```go
ts=2022-08-17T23:43:55.773Z caller=main.go:43 level=info msg="Starting minecraft_exporter" version="(version=0.16.0, branch=HEAD, revision=df4d15fe643b0104a0729a08f5340ff3285a6c9d)"
ts=2022-08-17T23:43:55.773Z caller=main.go:44 level=info msg="Build context" build="(go=go1.19, user=goreleaser, date=2022-08-09T05:58:33Z)"
ts=2022-08-17T23:43:55.774Z caller=main.go:53 level=info msg="Disabling collection of exporter metrics (like go_*)" value=true
ts=2022-08-17T23:43:55.774Z caller=main.go:81 level=info msg="Listening signals..."
ts=2022-08-17T23:43:55.774Z caller=main.go:69 level=info msg="Listening on address" address=:9150
ts=2022-08-17T23:43:55.774Z caller=tls_config.go:195 level=info msg="TLS is disabled." http2=false
ts=2022-08-17T23:44:49.024Z caller=exporter.go:897 level=error msg="Failed to get player stats" err=EOF
panic: runtime error: index out of range [1] with length 0

goroutine 39 [running]:
github.com/minecraft-exporter/pkg/exporter.(*Exporter).getServerStats(0xc000128240, 0xc000230040?)
        /home/runner/work/minecraft-prometheus-exporter/minecraft-prometheus-exporter/pkg/exporter/exporter.go:786 +0xb52
github.com/minecraft-exporter/pkg/exporter.(*Exporter).Collect(0xc000128240, 0xc00038a760?)
        /home/runner/work/minecraft-prometheus-exporter/minecraft-prometheus-exporter/pkg/exporter/exporter.go:900 +0x2c5
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
        /home/runner/go/pkg/mod/github.com/prometheus/client_golang@v1.13.0/prometheus/registry.go:453 +0x10d
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
        /home/runner/go/pkg/mod/github.com/prometheus/client_golang@v1.13.0/prometheus/registry.go:545 +0xbac
```

### How can we reproduce this?

Based on README, I've created following docker-compose which I used to run exporter:

```yaml
version: '3'

services:
  mmad-exporter:
    container_name: mmad-exporter
    image: ghcr.io/dirien/minecraft-exporter
    restart: unless-stopped
    networks:
      - pterodactyl_nw
    volumes:
      - /var/lib/pterodactyl/volumes/cebf950d-e27a-4a7a-a2d5-9d5952c7bb08/world:/minecraft/world:ro
    environment:
      - WEB_DISABLED_EXPORTER_METRICS=true
      - MC_RCON_ADDRESS=${SERVER}:25665
      - MC_RCON_PASSWORD=${RCON_PASS}
      - MC_NAME_SOURCE=mojang
      - MC_MOD_SERVER_STATS=forge
    ports:
      - 9150:9150
    expose:
      - 9150

networks:
  pterodactyl_nw:
    external:
      name: pterodactyl_nw
```

[part] prometheus.yml
```yaml
  - job_name: minecraft_exporter
    scrape_interval: 1m
    scrape_timeout: 45s
    static_configs:
      - targets: ["#REDACTED-IP#:9150"]
```

### minecraft-exporter --version

```bash
v0.16.0
```

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dirien** [comment]
hi @Kamisoi,

thanks for reporting the issue! Looks like there is a space in the Overall Tick RCON output:

```bash
[07:41:07] [Server thread/INFO] [minecraft/DedicatedServer]: Overall : Mean tick time: 5.203 ms. Mean TPS: 20.000
```

The version, I tested with (1.16) has no space:

```bash
07:48:49] [Server thread/INFO] [minecraft/DedicatedServer]: Overall: Mean tick time: 0,441 ms. Mean TPS: 20,000
```

I fixed it, with v0.16.1.

Looking forward for your feedback!

**Comment by dirien** [comment]
Created an issue #275 for the TPS from the dimensions.
